packages/management/tkr-service/README.md

```diff
@@ -0,0 +1,14 @@
+# featuregates Package
+
+Featuregates management package provides Features and Featuregates APIs to
```

Comment:
The readme needs to be updated with `tkr-service` instead of `Featuregates`.

Reply:
Thank you for catching this!